lib/samples/query_related_features/query_related_features.dart

```diff
@@ -0,0 +1,280 @@
+        : Container(
+            constraints: BoxConstraints(
+              maxHeight: MediaQuery.sizeOf(context).height * 0.4,
+            ),
+            padding: EdgeInsets.fromLTRB(
+              20.0,
+              5.0,
+              20.0,
+              max(
+                20.0,
+                View.of(context).viewPadding.bottom /
+                    View.of(context).devicePixelRatio,
+              ),
+            ),
+            width: double.infinity,
+            child: Column(
+              mainAxisSize: MainAxisSize.min,
+              children: [
+                Text(
+                  _selectedParkName,
+                  style: Theme.of(context).textTheme.titleLarge,
+                ),
+                const Divider(),
+                Text(
+                  'Alaska National Preserves',
+                  style: Theme.of(context).textTheme.titleMedium,
+                ),
+                const Divider(),
+                // Display the list of feature preserves for the selected park.
+                Flexible(
+                  child: ListView.builder(
+                    shrinkWrap: true,
+                    padding: EdgeInsets.zero,
+                    itemCount: _featurePreserves.length,
+                    itemBuilder: (context, index) {
+                      return ListTile(
+                        dense: true,
+                        contentPadding: EdgeInsets.zero,
+                        title: Text(
+                          _featurePreserves[index],
+                          style: Theme.of(context).textTheme.titleSmall,
+                        ),
+                      );
+                    },
+                  ),
+                ),
+                const Divider(),
+                Text(
+                  'Alaska National Species',
+                  style: Theme.of(context).textTheme.titleMedium,
+                ),
+                const Divider(),
+                // Display the list of feature species for the selected park.
+                Flexible(
+                  child: ListView.builder(
+                    shrinkWrap: true,
+                    padding: EdgeInsets.zero,
+                    itemCount: _featureSpecies.length,
+                    itemBuilder: (context, index) {
+                      return ListTile(
+                        dense: true,
+                        contentPadding: EdgeInsets.zero,
+                        title: Text(
+                          _featureSpecies[index],
+                          style: Theme.of(context).textTheme.titleSmall,
+                        ),
+                      );
+                    },
+                  ),
+                ),
+              ],
+            ),
+          );
+  }
+
+  void onMapViewReady() async {
+    // Create a map with a topographic basemap style.
+    final map = ArcGISMap.withBasemapStyle(BasemapStyle.arcGISTopographic);
+
+    // Feature table for the Alaska National Parks layer
```

Comment:
periods at the end of these comments

Reply:
And a few that need a caps on the first word